pom.xml

```diff
@@ -38,17 +38,18 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
+        <jenkins.version>2.60</jenkins.version>
+        <java.level>8</java.level>
```

Comment:
Its going to take years for people to be able to use any changes in this plugin from now onwards

![picture1](https://user-images.githubusercontent.com/50156/27812854-4b5524d2-60b5-11e7-96e8-fc3544028f9e.png)